Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1453,7 +1454,11 @@ bool MainWindow::eventFilter(QObject* object, QEvent* event)
   if (event->type() == QEvent::Close)
   {
     if (RequestStop() && object == this)
+    {
       m_exit_requested = true;
+      // If shutting down the emulator, we want to ignore breakpoints.
+      PowerPC::debug_interface.ClearAllBreakpoints();
```

Comment:
This should be done in RequestStop before the core is unpaused -- otherwise this will race with the CPU thread.